scripts/local_ossl_instl.sh

```diff
@@ -1,68 +1,55 @@
 #!/bin/bash
-set -e
+set -ex
 
-if [ "$#" -ne 1 ]; then
+if [ "$#" -eq 0 ]; then
     echo -e "Usage: ./local_ossl_instl.sh <OPENSSL_DIR>"
     exit 1
 fi
 
 echo "Setup for OpenSSL version 3.1.0 with FIPS module"
-echo "Installing OpenSSL to "${1}" ..."
+echo "Installing OpenSSL to ${1} ..."
 
 if [[ ! -d "${1}" ]]; then
     echo "ERROR: Specified directory does not exist"
     exit 2
 fi
 
-cd "${1}"
-export OPENSSL_DIR="$(pwd)"
-cd -
+pushd "$1"
+OPENSSL_DIR=$(pwd)
```

Comment:
```suggestion
OPENSSL_DIR="$(pwd)"
```
I'd suggest using expression quoting eveyrwhere for security reasons.

Reply:
OK